### ISSUE TITLE
Add default_target_group_arn_suffix Output

### ISSUE THREAD

**Issue by razorsedge** [cloudposse/terraform-aws-alb#105]
## Describe the Feature
Add `default_target_group_arn_suffix` as an output.

## Expected Behavior
Simplified Terraform code.

## Use Case
We use `cloudposse/alb-target-group-cloudwatch-sns-alarms/aws` along with this module and have to use a `data` source to build the `target_group_arn_suffix` input to that module.  By having this module output the data, there should be no need for the data source.

```
data "aws_lb_target_group" "mytg" {
  arn = module.alb.default_target_group_arn
}

module "alb_alarms" {
  source  = "cloudposse/alb-target-group-cloudwatch-sns-alarms/aws"
  version = "0.15.0"

  alb_arn_suffix          = module.alb.alb_arn_suffix
  target_group_arn_suffix = data.aws_lb_target_group.mytg.arn_suffix # This can be simplified.
}
```

## Describe Ideal Solution
```
output "default_target_group_arn_suffix" {
  description = "The default target group ARN suffix"
  value       = join("", aws_lb_target_group.default.*.arn_suffix)
}
```

**Comment by shinenelson** [comment]
I have the same issue myself. I came here to suggest the same thing.

How I dealt with it was to use the `aws_arn` data source

```terraform
data "aws_arn" "alb_target_group" {
  arn = module.alb.default_target_group_arn
}

module "alb_alarms" {
  source  = "cloudposse/alb-target-group-cloudwatch-sns-alarms/aws"
  version = "0.15.0"

  alb_arn_suffix          = module.alb.alb_arn_suffix
  target_group_arn_suffix = trimeprefix(data.aws_arn.alb_target_group.resource, "/targetgroup")
}
```

Regardless of the means, an output from the module would save us the need to have to use an extra data source when combining with `alb-target-group-cloudwatch-sns-alarms`.

Part of the reason I love using the `cloudposse` modules is the interoperability of their modules. Everything can be mixed-and-matched and will still work. I have wrapped multiple of cloudposse's modules to form modules myself bringing upto 3-4 levels of module nesting.

Let me see if I can put up a pull request for this soon.
